commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java

```diff
@@ -157,6 +157,7 @@
     <li class=fixed-in-release>Added support for {@link io.sphere.sdk.extensions.AuthorizationHeaderAuthentication} for api extensions.</li>
     <li class=fixed-in-release>Fixed {@link io.sphere.sdk.orderedits.commands.stagedactions.AddLineItem} staged update action structure</li>
     <li class=new-in-release>Added support for {@link io.sphere.sdk.stores.Store}</li>
+    <li class=new-in-release>Added support commands that allow you to access carts and orders belonging to a specific store.</li>
```

Comment:
Here it would be mice to add links to the newly added commands/requests (e.g. `CartInStoreCreateCommand`). I know that it's a lot of work, but this would make the changelog more useful for our users: They need some hints how we name these commands/requests, because it's a new convention and isn't part of our trainings yet.